mcs/class/referencesource/mscorlib/system/reflection/bindingflags.cs

```diff
@@ -64,5 +64,6 @@ public enum BindingFlags
 
         // These are a couple of misc attributes used
         IgnoreReturn        = 0x01000000,    // This is used in COM Interop
+        DoNotWrapExceptions = 0x02000000, // Disables wrapping exceptions in TargetInvocationException
```

Comment:
Please instead of changing this file replace it in `corlib.dll.sources` with the version from `external/corefx/src/Common/src/CoreLib/System/Reflection/BindingFlags.cs`

Reply:
Done